README.md

```diff
@@ -4,172 +4,42 @@ This repository holds the code for [wikispeedruns.com](https://wikispeedruns.com
 
 ## 1. Prerequisites
 
-- Python 3.7 or greater
```

Comment:
Maybe we can move this to a separate ReadMe linked from this main one. I think it's still helpful to have for those who aren't familar with Docker, as well as essentially documentation for what Docker is doing.

Reply:
There are also some optional steps to setup features like achievements and such that could be useful still.